// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`local-json-server > local-json-server test case 1`] = `
"──────────────────────────────────────────────────────────────────────────────── 
 
  Running workflow local-json-server.arazzo.yaml / before 
 
  ✗ GET /items - step get-all-items-before 
    ✗ status code check
 
──────────────────────────────────────────────────────────────────────────────── 
 
  Running workflow local-json-server.arazzo.yaml / crud-items 
 
  ✗ POST /items - step add-item 
    ✗ success criteria check - $statusCode == 201
    ✗ status code check
    ✗ unexpected error
 
──────────────────────────────────────────────────────────────────────────────── 
 
  Running workflow local-json-server.arazzo.yaml / auto-inherit 
 
  ✗ POST /items - step add-item-auto-inherit 
    ✗ status code check
    ✗ unexpected error
 

  Failed tests info:

  Workflow name: before
 
    stepId - get-all-items-before 
    ✗ status code check 
      
      
  Workflow name: crud-items
 
    stepId - add-item 
    ✗ success criteria check 
      Checking simple criteria: {"condition":"$statusCode == 201"}
       
    ✗ status code check 
      
       
    ✗ unexpected error 
    Reason: Error in resolving runtime expression '$response.body#/id'. 
    This could be because the expression references a value from a previous failed step, or is trying to reference a variable that hasn't been set.
  Workflow name: auto-inherit
 
    stepId - add-item-auto-inherit 
    ✗ status code check 
      
       
    ✗ unexpected error 
    Reason: Error in resolving runtime expression '$response.body#/id'. 
    This could be because the expression references a value from a previous failed step, or is trying to reference a variable that hasn't been set. 
  Summary for local-json-server.arazzo.yaml
  
  Workflows: 3 failed, 3 total
  Steps: 3 failed, 3 total
  Checks: 6 failed, 6 total
  Time: <test>ms 
 
 
┌───────────────────────────────────────────────────────────────────────┬────────────┬─────────┬─────────┬──────────┐
│ Filename                                                              │ Workflows  │ Passed  │ Failed  │ Warnings │
├───────────────────────────────────────────────────────────────────────┼────────────┼─────────┼─────────┼──────────┤
│ x local-json-server.arazzo.yaml                                       │ 3          │ 0       │ 3       │ -        │
└───────────────────────────────────────────────────────────────────────┴────────────┴─────────┴─────────┴──────────┘
 

 Tests exited with error 

"
`;
